sysutils/upt/Portfile

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                upt
+version             0.8
+revision            0
+
+platforms           darwin
+supported_archs     noarch
+categories          sysutils python
```

Comment:
you're right here... I forgot that the python PortGroup already sets the category to "python". Since it's in the "sysutils" category now that needs to be the first category mentioned. So this is fine, or you could use ```categories-prepend sysutils```. Please do put the field above ```platforms``` though as would be done in the ```base.Portfile```.